use-shopping-cart/src/@types/use-shopping-cart/index.d.ts

```diff
@@ -0,0 +1,105 @@
+	 * @param cancelUrl The redirect url for a cancelled sale
+	 * @param currency The preferred currency
+	 * @param language The language to be used in the checkout
+	 * @param billingAddressCollection Should the billing address be collected at the checkout. Defaults to false
+     * @param allowedCountries The allowed countries
+	 */
+    export const CartProvider: (props: ProviderProps) => JSX.Element;
+
+    export interface Product {
+        /** 
+         * The name of the product
+         */
+        name?: string;
+        /**
+         * The product sku
+         */
+        sku: string;
+        /**
+         * The price of the product
+         */
+        price: number;
+        /**
+         * An image of the product
+         */
+        image?: string;
+        /**
+         * The currency of the product
+         */
+        currency?: string;
+    }
+
+    export type CartDetails = {
+        [key: string]: any
+    };
+
+    export interface ShoppingCartUtilities {
+        /**
+         * Add an item to the cart
+         * @param product The product to add to the cart
+         */
+        addItem: (product: Product) => void, 
+        /**
+         * Remove a cart item
+         * @param sku The item to remove sku
+         */
+        removeCartItem: (sku: string) => void,
+        /**
+         * Reduce the quantity of items by one in the cart
+         * @param sku The sku of the item to reduce quantity by one
+         */
+        reduceItemByOne: (sku: string) => void,
+        /**
+         * Calculates the total price of the cart items
+         */
+        totalPrice: () => string;
+        /**
+         * The number of items in the cart
+         */
+        cartCount: (items: number) => void;
+        /**
+         * Cart details is an object with skus of the items in the cart as keys and details of the items as the value,
+         */
+        cartDetails: (cartEntries: CartDetails) => void;
+        /**
+         * Redirects customers to the Stripe checkout
+         * @returns result object || error message
+         */
+        redirectToCheckout: (sessionId?: string) => Promise<any>;
+        /**
+         * Totally clears the cart of all items
+         */
+        clearCart: () => void;
+    }
+
+    /**
+     * Provides several utilities and pieces of data for you to use in your application.
+     * @returns addItem - Add an item to the cart
+     * @returns removeCartItem - Remove a cart item
+     * @returns reduceItemByOne - Reduce the quantity of items by one in the cart
+     * @returns totalPrice - Calculates the total price of the cart items
+     * @returns cartCount - The number of items in the cart
+     * @returns cartDetails - An object of skus
+     * @returns redirectToCheckout - Redirects customers to the Stripe checkout
+     * @returns clearCart - Totally clears the cart of all items
+     */
+    export declare function useShoppingCart (): ShoppingCartUtilities;
```

Comment:
We're also exporting `toCurrency` from index.js as well

Reply:
Oh, i missed that one, will add it

Reply:
I also just renamed `toCurrency` to `formatCurrencyString` per #74 


(sorry lol)

Reply:
Yeah i saw they were different in the docs and code. Will change that :)

Reply:
`toCurrency` needs to be updated to `formatCurrencyString` still